kinetic/distribution.yaml

```diff
@@ -1519,7 +1519,17 @@ repositories:
       type: git
       url: https://github.com/ethz-asl/grid_map.git
       version: master
-    status: developed
+    status: developed  
+  haf_grasping:
+    doc:
+      type: git
+      url: https://github.com/davidfischinger/haf_grasping.git
+      version: kinetic
+    source:
+      type: git
+      url: https://github.com/davidfischinger/haf_grasping.git
+      version: kinetic
+    status: maintained    
```

Comment:
This line as well as the other `status` line contain trailing whitespaces which need to be removed.